src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -31,7 +34,10 @@
 cli_command(__name__, 'network application-gateway list', custom_path.format('list_application_gateways'))
 cli_command(__name__, 'network application-gateway start', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.start', cf_application_gateways)
 cli_command(__name__, 'network application-gateway stop', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.stop', cf_application_gateways)
-cli_command(__name__, 'network application-gateway show-backend-health', 'azure.mgmt.network.operations.application_gateways_operations#ApplicationGatewaysOperations.backend_health', cf_application_gateways)
+
+if get_api_version(ResourceType.MGMT_NETWORK) in ['2016-09-01']:
```

Comment:
Batch has an API version of 2017-01-01.4.0 and then there are preview APIs, and some preview APIs are actually GA, etc.